xcdat/dataset.py

```diff
@@ -162,145 +169,128 @@ def open_mfdataset(
 
     Keep a single variable in the Dataset:
 
-    >>> from xcdat.dataset import open_dataset
+    >>> from xcdat.dataset import open_mfdataset
     >>> ds = open_mfdataset(["file_path1", "file_path2"], data_var="tas")
-
-    Keep multiple variables in the Dataset:
-
-    >>> from xcdat.dataset import open_dataset
-    >>> ds = open_mfdataset(["file_path1", "file_path2"], data_var=["ts", "tas"])
     """
-    # check if time axis is cf_compliant
-    cf_compliant = _check_dataset_for_cf_compliant_time(paths)
-
-    # if cf_compliant, let xarray decode the time units
-    # otherwise, decode using decode_time_units
-    if cf_compliant:
-        ds = xr.open_mfdataset(paths, decode_times=True, data_vars=data_vars, **kwargs)
+    if decode_times:
+        cf_compliant_time: Optional[bool] = has_cf_compliant_time(paths)
+        if cf_compliant_time is True:
+            ds = xr.open_mfdataset(
+                paths, decode_times=True, data_vars=data_vars, **kwargs
+            )
+        elif cf_compliant_time is False:
+            ds = xr.open_mfdataset(
+                paths, decode_times=False, data_vars=data_vars, **kwargs
+            )
+            ds = decode_non_cf_time(ds)
```

Comment:
FIXME: There is still the issue of datasets with time coordinates that have non-CF compliant time units, each with a different reference date.